bot/kodiak/queue.py

```diff
@@ -18,7 +19,7 @@
 logger = structlog.get_logger()
 
 
-MERGE_QUEUE_NAMES = "kodiak_merge_queue_names"
+MERGE_QUEUE_NAMES = "kodiak_merge_queue_names:v2"
```

Comment:
With this change we ignore the existing merge queues on start since they old names are no longer used.